ci/build.yaml

```diff
@@ -0,0 +1,25 @@
+---
+platform: linux
+
+image_resource:
+  type: registry-image
+  source:
+    repository: golang
+    tag: 1.25.2-bullseye
```

Comment:
This one's quite out of date (1.25.5 is out) and is not bookworm either. Also do we still need a 1.24 version for this?